core/src/toga/types.py

```diff
@@ -13,6 +13,10 @@
 
     PositionT: TypeAlias = toga.Position | tuple[int, int]
     SizeT: TypeAlias = toga.Size | tuple[int, int]
+else:
+    # sphinx-autodoc-typehints errors when these are gated with TYPE_CHECKING...
+    PositionT = None
+    SizeT = None
```

Comment:
Any idea why it only affects PositionT and SizeT, and not any of the other type aliases?

Reply:
I didn't look in to it too far yet....but I think it could be related to the circular import